hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/CompareIndexService.kt

```diff
@@ -69,7 +69,7 @@ class CompareIndexService(
     }
   }
   fun compareIndex(): Pair<List<String>, List<String>> {
-    val allNomis = nomisService.getPrisonerNumbers(0, Int.MAX_VALUE).sorted()
+    val allNomis = nomisService.getPrisonerNumbers(0, 10000000).sorted()
```

Comment:
this was being blocked by the new Prison API cloud platform firewall.  No reason to have it so high anyway